### PR TITLE
Ipfabric ssl verification

### PR DESCRIPTION
**Pull request by justinjeffery-ipf**
Added SSL verification for IP Fabric to integrate with self-signed instances.

### CODE REVIEW

**Comment by justinjeffery-ipf** [comment]
#65

**Review by chadell — APPROVED** [review]
LGTM

File: development/nautobot_config.py [hunk 1]
```diff
@@ -310,6 +310,7 @@ def is_truthy(arg):
     "nautobot_chatops_ipfabric": {
         "IPFABRIC_API_TOKEN": os.environ.get("IPFABRIC_API_TOKEN"),
         "IPFABRIC_HOST": os.environ.get("IPFABRIC_HOST"),
+        "IPFABRIC_VERIFY": os.environ.get("IPFABRIC_VERIFY", True),
```
Comment (pke11y) [inline development/nautobot_config.py:313]: Could you wrap that with the `is_truthy` function for safety.
```suggestion
        "IPFABRIC_VERIFY": is_truthy(os.environ.get("IPFABRIC_VERIFY", True)),
```

**Review by pke11y — CHANGES_REQUESTED** [review]
Looks great! Thanks for the PR. One suggestion.

**Review by pke11y — APPROVED** [review]
Nice job. Thanks!

**Pull request merged**